freeipa_auth/backends.py

```diff
@@ -128,9 +132,9 @@ def update_user_groups(self, user, groups):
 
         # Update user groups
         if self.settings.UPDATE_USER_GROUPS:
+            all_groups = Group.objects.all()
+            user.groups.remove(*all_groups)
```

Comment:
I _believe_ that we could get away with `user.groups.clear()` ([docs](https://docs.djangoproject.com/en/4.1/ref/models/relations/#django.db.models.fields.related.RelatedManager.clear))

Reply:
_i guess andrew beat me to it..._

Reply:
updated